style.css

```diff
@@ -0,0 +1,70 @@
+html{
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Helvetica Neue', Arial, sans-serif;
+}
+
+body{
+    font-size: 20px;
+    color: #333;
+    background-color: white;
+    padding: 1.4em 0.15em;
+}
+
+.main-container{
+    padding: 1rem 0.5rem;
+    border-radius: 0.5em;
+    background-color: #eeeeee;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.main-container > h1 {
+    margin-bottom: 0;
+}
+
+.main-container > p {
+    font-size: 16px;
+}
+
+.image-container{
+    background-color: #fff;
+    width: 75%;
+    display: flex;
+    flex-direction: column;
+    padding: 0 12.5%;
+}
+
+.image-container > p{
+    font-size: 16px;
+}
+
+.tribute-info > li{
+    display: flex;
+    
+}
+
+li{
+    margin: 20px;
+    font-size: 1em;
+
+}
+span{
+    font-weight: bold;
+}
+
+.tribute-info{
+    align-items: center;
+    justify-content: center;
+    display: flex;
+    flex-direction: column;
+    width: 45%;
+    padding-top: 32px;
+
+}
+
+.tribute-paragraph{
+    font-style: italic;
+    padding-left: 36px;
+}
+
```

Comment:
- [ ] Please do make sure your eslint has no errors. The code will look prettier.